src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesView/ExternalDataSourcesView.js

```diff
@@ -288,8 +288,9 @@ const ExternalDataSourcesView = ({
         }}
         mutators={{ ...arrayMutators }}
         onDelete={onDelete}
-        onSubmit={values => {
+        onSubmit={(values, form) => {
           onSubmit(values);
+          form.reset();
```

Comment:
I'm not sure about this... Does ensuring the initialValues state up above is correct not fix the issue?


If not, then I accept this solution, but would like a comment explaining why it's needed, and perhaps an issue in the stripes-kint-components gitlab for me to dig into what happens on FormModal submit. I'd rather not have to repeat this fix everywhere we use formModal

Reply:
> I'm not sure about this... Does ensuring the initialValues state up above is correct not fix the issue?
No. The initialValues contain the correct data, but the field is still filled with the data from previous create resp. edit (when edit was not successful)